hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -175,6 +175,14 @@ private Message process(final SCMRatisRequest request) throws Exception {
     }
   }
 
+  @Override
+  public void notifyLogFailed(Throwable ex,
+                              RaftProtos.LogEntryProto failedEntry) {
+    // Should we add retry here?
```

Comment:
it is better to remove this comment, since io error of appending log is a fatal error and not recoverable. we just terminate the process